README.md

```diff
@@ -3,7 +3,7 @@
 GROQ-JS is a JavaScript implementation of [GROQ](https://www.sanity.io/docs/data-store/how-queries-work) which follows the official specification.
 
 ```javascript
-import {parse, evaluate} from 'groq-js/v2'
+import {parse, evaluate} from 'groq-js/1'
```

Comment:
Should we add a line or two explaining the need to import based on GROQ versions? I know we don't have a V2 yet, but we can explain how it works.